dcsazure_adls_to_adls_mask_pl/README.md

```diff
@@ -0,0 +1,75 @@
+# dcsazure_adls_to_adls_mask_pl
+## Delphix Compliance Services (DCS) for Azure - ADLS to ADLS Masking Pipeline
+
+This pipeline will perform masking of data from your Azure Data Lake (ADLS) Data from one location to another.
+
+### Prerequisites
+
+1. Configure the hosted metadata database and associated Azure SQL service (version `V2024.04.18.0`+).
+1. Configure the DCS for Azure REST service.
+1. Configure the Azure Data Lake Storage service associated with your ADLS source data.
+1. Configure the Azure Data Lake Storage service associated with your ADLS sink data.
```

Comment:
are these supposed to be all 1's?

Reply:
Yep. Markdown is smart enough to render them as incremental numbers, you can see that via the rich diff. Which is very handy if you need to add something in the middle of the list later, you don't have to go back and re-number everything.

Even in comments, the following:
```
1. This
1. Is
1. An
1. Example
```

Is rendered as follows:

1. This
1. Is
1. An
1. Example